Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -126,8 +126,7 @@ public override void Destroy()
         /// <summary>
         /// The <see cref="GameObject"/> to which observed objects are parented.
         /// </summary>
-        private GameObject ObservedObjectParent => observedObjectParent ?? (observedObjectParent = SpatialAwarenessSystem?.CreateSpatialAwarenessObjectParent("WindowsMixedRealitySpatialMeshObserver"));
-
+        private GameObject ObservedObjectParent => observedObjectParent != null ? observedObjectParent : (observedObjectParent = SpatialAwarenessSystem?.CreateSpatialAwarenessObjectParent("WindowsMixedRealitySpatialMeshObserver"));
```

Comment:
why this change? isn't the original version the pattern established for MRTK?

Reply:
The difference here is that most of those patterns are referring to interfaces, while this one is a `GameObject`, and Unity uses a custom equality check for null which doesn't work in this case.